config/locales/en.yml

```diff
@@ -974,6 +974,58 @@ en:
   community_id_is_precise: Community Taxon is precise
   community_moderation_settings: Community Moderation Settings
   community_supported_id: Community-supported Taxon?
+  compact_number_formatting:
+    x_thousand:
+      one: "%{count}K"
+      other: "%{count}K"
+    # Short notation representing a four-digit number, i.e. 1,000
+    x_ten_thousand:
+      one: "%{count}K"
+      other: "%{count}K"
```

Comment:
I'm sure I missed some context about this in kickoff, a couple questions:

1. Can you provide some examples of why we need to pluralize these? We already have some of these under [`number.format.si`](https://github.com/inaturalist/inaturalist/blob/main/config/locales/en.yml#L7719) so if we don't need to pluralize we should probably build on those. If we do need to pluralize, then this separate set of abbreviations is probably gustified.
1. Why are you using the `one` and `other` and not `few` and `many`?
1. Isn't `x_ten_thousand` supposed to be for a five-digit number, e.g. 10,000? I think a lot of these context comments are off.